terracumber_config/tf_files/SUSEManager-4.2-build-validation-NUE.tf

```diff
@@ -599,24 +580,9 @@ module "debian10-minion" {
   ssh_key_path            = "./salt/controller/id_rsa.pub"
 }
 
-module "debian11-minion" {
-  source             = "./modules/minion"
-  base_configuration = module.base_core.configuration
-  product_version    = "4.2-released"
-  name               = "min-debian11"
-  image              = "debian11o"
-  provider_settings = {
-    mac                = "aa:b2:92:42:00:6e"
-    memory             = 4096
-  }
+// Debian 11 is not supported by SUSE Manager 4.2
 
-  server_configuration = {
```

Comment:
Debian 12 might not be supported but isn't this the _server_ configuration? don't we need this?

Reply:
nevermind, this is inside the debian 11 module, my bad

Reply:
this "server_configuration" is a pointer to the server's configuration used by this minion

I agree this is a terrible name